test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -30,9 +30,13 @@ export default class PossibleIssues extends Component {
         if (response.ok) {
             const relatedIssues = await response.json();
             let dataSource = [];
+            const repoUrlAPIPrefix = "https://api.github.com/repos/";
             for (let index = 0; index < relatedIssues.items.length; index++) {
+                const repoName = relatedIssues.items[index].repository_url.replace(repoUrlAPIPrefix, "");
+                const repoUrlwithIssues = `https://github.com/${repoName}/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+${testName}`;
                 dataSource.push({
                     key: index,
+                    repos: <a href={repoUrlwithIssues} target="_blank" rel="noopener noreferrer">{repoName}</a>,
```

Comment:
I think there is no need to repeat repo per issue. For example, if we have 10 issues in one repo, this repo will be repeated 10 times. 

I think we should do one repo per table.

|OpenJ9|
|--|
|issue123|
|issue456|

|AdoptOpenJDK/Infra|
|--|
|issueABC|
|issueXYZ|

`dataSource` contains multiple obj arrays. One obj array per repo.
In render(), loop `dataSource`, and display one obj array per table.

Reply:
The new commit has been modified to display one repo per table as suggested. Could you check it? Thanks.
![image](https://user-images.githubusercontent.com/48260423/86812894-86837b80-c04d-11ea-8598-dd0813df716d.png)